modules/notes.php

```diff
@@ -104,7 +104,16 @@ function action_init() {
 	}
 
 	function styles_and_scripts() {
-		if ( !is_rtl() ) {
+		/**
+		 * Can be used to force Notifications to display in RTL style.
+		 *
+		 * @since 4.7.0
+		 *
+		 * @param bool true Should notifications be displayed in RTL style. Defaults to false.
+		 */
+		$enqueue_rtl = apply_filters( 'enqueue_rtl_notification_styles', false );
```

Comment:
Good point, prefixed with `a8c_wpcom_masterbar_` in 0660ea26a58439a22995179e04b38be434d821f6.